YellowNotes.Api/Controllers/UserController.cs

```diff
@@ -0,0 +1,58 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using YellowNotes.Core.Dtos;
+using YellowNotes.Core.Services;
+namespace YellowNotes.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
```

Comment:
That's not really the right endpoint we want here. It should be `/users`.

Reply:
You mean `api/users`, right?

Reply:
The `api` is not really needed here because backend will run on a different port.